packages/cw20/src/logo.rs

```diff
@@ -0,0 +1,39 @@
+use cosmwasm_std::Binary;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+/// This is used for uploading logo data, or setting it in InstantiateData
+#[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
+#[serde(rename_all = "snake_case")]
+pub enum Logo {
+    /// A reference to an externally hosted logo. Must be a valid HTTP or HTTPS URL.
+    /// PLEASE REVIEW: is this dangerous from a security point of view?
```

Comment:
I don't think it is insecure in anyway from blockchain POV - we are not using it internally. The question is if it is possible to use it on external side, and I am not aware about remote execution method using embedded images. And even if it is a case, I think it is webpage who is using this URL responsibility to ensure this image is trusted. My concern is, that keeping images in blockchain/external servers allows abuses with usage of logo, as it is more difficult to moderate this if it lies in blockchain than if it is on owner server, but this relates to both methods of uploading image.

Anyway - whatever final decision is, pleas remember to remove this comment before merging.